src/components/chapters/Info/InfoPage.tsx

```diff
@@ -28,6 +28,7 @@ const InfoPage: React.FC<Props> = ({ hasError, chapterInfoResponse, chapterRespo
           lang,
         )}-${toLocalizedNumber(chapterResponse.chapter.versesCount, lang)}`}
         canonical={getCanonicalUrl(lang, getSurahInfoNavigationUrl(chapterResponse.chapter.slug))}
+        description={chapterInfoResponse.chapterInfo.shortText}
```

Comment:
checked a bit, looks like the desc is not in in /surah-info/1 and surah/2/info. will send more into to discord